src/iperf_api.c

```diff
@@ -2498,6 +2538,8 @@ iperf_reset_test(struct iperf_test *test)
 
     SLIST_INIT(&test->streams);
 
+    free(test->remote_congestion_used);
+    test->remote_congestion_used = NULL;
```

Comment:
I suggest:
`if (test->remote_congestion_used)`
`free(test->remote_congestion_used);`

Reply:
free(NULL) is always safe to do as per the man pages. I don't see the benefit of doing the if for the extra lines and risk of a typo between the 2.

Reply:
You have a perfectly valid point about `free(NULL)` being safe to do.  However the rest of iperf3 does this sort of check...I want to try to keep some kind of consistency in the code idioms within iperf3, even if they might be suboptimal.